mne/tests/test_epochs.py

```diff
@@ -3201,6 +3287,17 @@ def test_drop_epochs():
     assert_array_equal(events[epochs[3:].selection], events1[[5, 6]])
     assert_array_equal(events[epochs["1"].selection], events1[[0, 1, 3, 5, 6]])
 
+    # Test using tuple to drop epochs
+    raw, events, picks = _get_data()
+    epochs_tuple = Epochs(raw, events, event_id, tmin, tmax, picks=picks, preload=True)
+    selection_tuple = epochs_tuple.selection.copy()
+    epochs_tuple.drop((2, 3, 4), reason=([["list"], "string", ("tuple",)]))
+    n_events = len(epochs.events)
+    assert_equal(
+        [epochs_tuple.drop_log[k] for k in selection_tuple[[2, 3, 4]]],
+        [["list"], ["string"], ["tuple"]],
+    )
```

Comment:
For native Python types you should be able to use pytest directly (it overwrites `assert` to be informative on error
```suggestion
    assert [epochs_tuple.drop_log[k] for k in selection_tuple[[2, 3, 4]]] == [("list",), ("string",), ("tuple",)]
```
Note that this is also a tiny bit more specific because it asserts that the entries are tuples, whereas `assert_equal` will treat list and tuple the same